### ISSUE TITLE
FCN.py文件中的错误

### ISSUE THREAD

**Issue by RobotMo** [fuyongXu/FCN_Pytorch_Simple#1]
make_layers函数中，in_channels变量的值应该是随着层数的推进，表示上一层的输出值，而不是固定为3. 如下所示，增加的一行位于**return前**
```python
def make_layers(cfg,batch_norm=False):
    layers = []
    in_channels = 3
    for v in cfg:
        if v == 'M':
            layers += [nn.MaxPool2d(kernel_size=2,stride=2)]
        else:
            conv2d = nn.Conv2d(in_channels,v,kernel_size=3,padding=1)
            if batch_norm:
                layers += [conv2d,nn.BatchNorm2d(v),nn.ReLU(inplace=True)]
            else:
                layers += [conv2d,nn.ReLU(inplace=True)]
            in_channels = v
    return nn.Sequential(*layers)
